src/structures/GuildChannel.js

```diff
@@ -293,7 +293,7 @@ class GuildChannel extends Channel {
         topic: data.topic,
         nsfw: data.nsfw,
         bitrate: data.bitrate || (this.bitrate ? this.bitrate * 1000 : undefined),
-        user_limit: data.userLimit != null ? data.userLimit : this.userLimit, // eslint-disable-line eqeqeq
+        user_limit: data.userLimit !== undefined ? data.userLimit : this.userLimit,
```

Comment:
Just asking, doesn't Discord.js code check undefined by `typeof variable === 'undefined'`? There's also another occurence in this same file.

Reply:
`!= null` checks for `!== null` and `!== undefined` @Dev-Yukine please put it back

Reply:
That's the point of this PR: To allow `null` to be used to remove the user limit.
Checking should probably be done with a typeof like kyra mentioned.

Reply:
zero removes the limit